srcalc/src/test/java/gov/va/med/srcalc/service/AdminServiceIT.java

```diff
@@ -204,4 +205,25 @@ public final void testGetAllSpecialties()
         // Ensure RiskModels are loaded.
         assertEquals(1, specList.get(0).getRiskModels().size());
     }
+    
+    @Test
+    public final void testUpdateCategories() throws Exception
```

Comment:
This test does not fail on master, so it doesn't actually verify that the bug was fixed. Please investigate.